src/online2/online-ivector-feature.h

```diff
@@ -486,7 +486,7 @@ class OnlineSilenceWeighting {
   // this output to class OnlineIvectorFeature by calling its function
   // UpdateFrameWeights with the output.
   void GetDeltaWeights(
-      int32 num_frames_ready_in,
+      int32 num_frames_ready_in, int32 frame_offset_in,
```

Comment:
Need to have some good documentation on what this new parameter does.  
Will the t values in the delta-weights be relative to this frame offset, or not?